src/pages/Facility/billing/account/components/PrintChargeItems.tsx

```
@@ -363,669 +367,719 @@ export const PrintChargeItems = (props: {
               </div>
             );
           })()}
+
+        <div className="gap-4 flex items-center flex-wrap">
+          <label className="text-sm whitespace-nowrap">
+            {t("filter_by_status")}:
+          </label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)